packages/at_secondary_server/lib/src/server/at_secondary_config.dart

```diff
@@ -5,11 +5,10 @@ import 'package:at_commons/at_commons.dart';
 import 'package:at_secondary/src/conf/config_util.dart';
 
 class AtSecondaryConfig {
-  static Map<ModifiableConfigs, ModifiableConfigurationEntry> _streamListeners =
-      {};
+  static final Map<ModifiableConfigs, ModifiableConfigurationEntry> _streamListeners = {};
   //Certs
-  static final bool? _useSSL = true;
-  static final bool? _clientCertificateRequired = true;
+  static final bool _useSSL = true;
```

Comment:
Nit - SSL is long deprecated vs TLS (though I realise we often still talk about 'SSL')

Reply:
Done